index.html

```diff
@@ -16,13 +16,50 @@
 
   <body>
     <div>
+      <nav class="coa-box-shadow">
```

Comment:
this while file is not super sustainable. at the very least we'd split these components up into snippets. 

There are lots of ways to do this: handlebars, Jinja, React, Svelte...I'm not worried about it for MVP. We can pick the best platform for templating once we have the design nailed down.